samples/jsonParserComparison.js

```diff
@@ -0,0 +1,165 @@
+const snowflake = require('snowflake-sdk');
+const helpers = require('./helpers');
+const blocked = require('blocked-at');
+
+async function run() {
+  console.log('Started with arguments: ');
+  console.log(`Inserted rows amount: ${process.argv[2]} - default 300000`);
+  console.log(`Selected rows amount: ${process.argv[3]} - default 300000`);
+  console.log(`Selected json parse : ${process.argv[4]} - default all of Function, eval, better-eval, JSON`);
+
+  const connection = await helpers.connectUsingEnv();
```

Comment:
let's connect separately for each test case

Reply:
ok